clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards/dptp.jsonnet

```diff
@@ -23,15 +23,15 @@ dashboard.new(
 .addPanel(
     (graphPanel.new(
         'CI Operator Failure Rates by Reason',
-        description='sum(rate(ci_operator_error_rate{state="failed",reason!~".*executing_template",reason!~".*executing_multi_stage_test",reason!~".*building_image_from_source",reason!~".*building_.*_image",reason!="executing_graph:interrupted"}[30m])) by (reason)',
+        description='sum(rate(ci_operator_error_rate{state="failed",reason!~".*cloning_source",reason!~".*executing_template",reason!~".*executing_multi_stage_test",reason!~".*building_image_from_source",reason!~".*building_.*_image",reason!="executing_graph:interrupted"}[30m])) by (reason)',
```

Comment:
err why do you not put it in the same regex by doing a `reason !~ "reason_a|reason_b"`? That would be a lot easier to read